### PR TITLE
TC02 - User cannot login with blank 'Username' text box

### PR DESCRIPTION
**Pull request by thienan090196**

### CODE REVIEW

File: src/main/java/com/logigear/helpers/Constant.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
     public static final String RAILWAY_URL = "http://www.railway2.somee.com/";
     public static final String EMAIL = "testrailway01@gmail.com";
     public static final String PASSWORD = "abc@12345";
+    public static final String BLANK_STRING = "";
```
Comment (tanmle) [inline src/main/java/com/logigear/helpers/Constant.java:8]: again, we don't need a constant for a blank value.

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/main/java/com/logigear/helpers/Constant.java (hunk 1 above)
Comment (thienan090196) [reply src/main/java/com/logigear/helpers/Constant.java:8]: I just removed the variable, I am uploading my latest code

File: src/test/java/com/logigear/tests/LoginTest.java [hunk 2]
```diff
@@ -23,4 +24,16 @@ public void TC01() {
...
+
+    @Test(description = "User cannot login with blank 'Username' text box")
+    public void TC02() {
+        account = new Account("", Constant.PASSWORD);
+        homePage.goToLoginPage();
+        loginPage.login(account);
+
+        String actualResult = loginPage.getErrorMessage();
+        String expectedResult = ErrorMessage.LOGIN_ERROR;
+        Assert.assertEquals(actualResult, expectedResult, "Error message fails to display!");
```
Comment (tanmle) [inline src/test/java/com/logigear/tests/LoginTest.java:36]: why do we 2 assertions with a same meaning?

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/test/java/com/logigear/tests/LoginTest.java (hunk 2 above)
Comment (thienan090196) [reply src/test/java/com/logigear/tests/LoginTest.java:36]: I removed the duplicate code and re-pushed the code. Please review for me

**Review by tanmle — APPROVED** [review]

**Pull request merged**